### ISSUE TITLE
Invalid gateway ID

### ISSUE THREAD

**Issue by jetchirag** [centralnicgroup-opensource/rtldev-middleware-blesta-ispapi-registrar#190]
**Describe the bug**
Unable to add hexonet account in blesta. Module installs successfully, but on entering account details it returns blank page. On check logs, I get this error:
```
[2024-06-22T15:29:22.826303+00:00] general.ERROR: Exception: a:1:{s:9:"module_id";a:1:{s:6:"exists";s:19:"Invalid gateway ID.";}} in /home/xyz/webapps/xyz/public/components/modules/ispapi/apis/BlestaLogger.php:77 Stack trace: #0 /home/xyz/webapps/xyz/public/components/modules/ispapi/apis/BlestaLogger.php(31): BlestaLogger->logHandle() #1 /home/xyz/webapps/xyz/public/components/modules/ispapi/apis/vendor/centralnic-reseller/php-sdk/src/HEXONET/Client.php(530): BlestaLogger->log() #2 /home/xyz/webapps/xyz/public/components/modules/ispapi/lib/Base.php(54): CNIC\HEXONET\Client->request() #3 /home/xyz/webapps/xyz/public/components/modules/ispapi/ispapi.php(2667): HEXONET\MODULE\LIB\Base->call() #4 [internal function]: Ispapi->validateConnection() #5 /home/xyz/webapps/xyz/public/vendors/minphp/input/src/Input.php(546): call_user_func_array() #6 /home/xyz/webapps/xyz/public/vendors/minphp/input/src/Input.php(367): Minphp\Input\Input->validateRule() #7 /home/xyz/webapps/xyz/public/components/modules/ispapi/ispapi.php(789): Minphp\Input\Input->validates() #8 /home/xyz/webapps/xyz/public/app/models/module_manager.php(567): Ispapi->addModuleRow() #9 /home/xyz/webapps/xyz/public/app/controllers/admin_company_modules.php(352): ModuleManager->addRow() #10 /home/xyz/webapps/xyz/public/vendors/minphp/bridge/src/Lib/Dispatcher.php(142): AdminCompanyModules->addRow() #11 /home/xyz/webapps/xyz/public/index.php(21): Dispatcher::dispatch() #12 {main}
```

**Context (please complete the following information):**
 - Device: Desktop
 - OS: Ubuntu 20
 - Browser Chrome
 - Blesta Version 5.8.1
 - PHP Version 8.1
 - ISPAPI Registrar Module Version 3.6.0

**Comment by KaiSchwarz-cnic** [comment]
We released a new version today. Do you mind trying if that one helps? It patched issues with 5.10, but maybe also this.

If not let us know.

**Comment by KaiSchwarz-cnic** [comment]
In addition, let me forward this guide to your hands: [WHMCS - Connectivity Issues](https://www.hexonet.support/hc/en-gb/articles/13834070620573-WHMCS-Connectivity-Issues).

It is related to WHMCS, but applies to Blesta as well. "Invalid Gateway Id" is not a common HTTP error we ever heard of. If you go through the steps in that guide, you'll be able to identify the root cause. Maybe some internal network issue / firewall / loadbalancer making trouble? ping our api and give some curl requests a try from your blesta server. If those work, then usually our blesta integration should be doing fine as well.

In parallel, I asked our responsible dev to check if your issue can be reproduced on our end.

**Comment by AsifNawaz-cnic** [comment]
Hi @jetchirag,

Thank you for reporting the issue.

I am pleased to inform you that the "Invalid gateway ID" issue has been fixed in our latest release, [v3.7.0](https://github.com/centralnicgroup-opensource/rtldev-middleware-blesta-ispapi-registrar/releases/tag/v3.7.0).

Please let me know if you have any questions or need further assistance.
